training/app.py

```diff
@@ -44,9 +44,11 @@ def first_page():
     from training.controllers.auth import bp
     from training.controllers.basic_funcionalities import bp as bp2
     from training.controllers.mail import bp as bp3
+    from training.controllers.api import bp as bp4
```

Comment:
Please format the code properly, there should be an empty line between imports and other statements

Reply:
Changed!